lms/static/sass/views/_login-register.scss

```diff
@@ -509,16 +509,16 @@
         }
 
         &.button-oa2-azuread-oauth2 {
-            color: darken($microsoft-blue, 20%);
+            color: darken($microsoft-black, 20%);
```

Comment:
@hamzamunir7300 Do we still need to darken this color? Should we just change this line to `color: $microsoft-black;`?